### ISSUE TITLE
there isn't a config.ini

### ISSUE THREAD

**Issue by kornface13** [rpesche/cryptopcash#5]
i don't use cryptop and your software doesn't generate a config.ini.  you should include a default one.

**Comment by rpesche** [comment]
Good point

**Comment by rpesche** [comment]
Here is a sample file.
I will add this to the package soon.

```ini
[theme]
# colors of the ui
# Available colors :
#   * black,
#   * yellow,
#   * white,
#   * dark red,
#   * light red,
#   * dark green,
#   * light green
#   * brown,
#   * dark blue,
#   * light blue,
#   * dark magenta,
#   * light magenta,
#   * dark cyan,
#   * light cyan,
#   * dark gray
#   * light gray
text=light green
banner=light green
banner_text=black

[api]
# Fiat currency for conversion
# Default value : USD
# Possible value : EUR USD GBP RUB ...
currency=EURO
```

**Comment by rpesche** [comment]
resolved in 1.1.0 version
